web/bfex/components/scraper/profile_scraper.py

```diff
@@ -3,8 +3,16 @@
 from bfex.components.scraper.scrapp import *
 
 class ProfileScraper(Scraper):
+    """ This is what will scrape the text off of the Faculty of Science
```

Comment:
So for docstrings, there should be a short description on the first line, that does not wrap. Then blank space, then you can have more text with further description, followed by arguments exceptions returns etc.